package.json

```diff
@@ -319,7 +321,7 @@
         "apl-viewhost-web": "1.4.0",
         "ask-smapi-sdk": "^1.2.2",
         "async-retry": "^1.3.1",
-        "axios": "^0.21.1",
+        "axios": "^0.19.2",
```

Comment:
You need to revert this back to 0.21.1 because of this security issue : https://github.com/alexa/ask-toolkit-for-vscode/pull/80

Reply:
My bad! I missed syncing my repo.